common/unified-chart/templates/high-priority/analysis-template.yaml

```diff
@@ -0,0 +1,28 @@
+{{- $top := . -}}
+{{- $values := .Values -}}
+{{- if .Values.argoRollouts.progressiveDelivery.enabled }}
+---
+apiVersion: argoproj.io/v1alpha1
+kind: AnalysisTemplate
+metadata:
+  name: black-box
```

Comment:
use include in every name
names cant be hardcoded 
will cause name collisions between services